vividus-extension-selenium/src/main/java/org/vividus/ui/session/AbstractSessionLinkPublisher.java

```diff
@@ -14,46 +14,44 @@
  * limitations under the License.
  */
 
-package org.vividus.bdd.browserstack.steps;
+package org.vividus.ui.session;
```

Comment:
reporter makes me thinking about report module and asking question why reporter thing does in selenium extension, but when we name the package session we like make a reference to selenium session which is suitable for the extension-selenium module

Reply:
no, the goal of this class is to provide the data for the report, as for session - we don't manage sessions, the session is just a data

>why reporter thing does in selenium extension
because it's a test automation solution, which includes reporting, logging and etc. and each module can report, log and etc.

Reply:
I didn't mean managing sessions, but what we do here is providing link to remote selenium session and this can be used not only for reporting